src/test/java/reposense/git/GitCheckerTest.java

```diff
@@ -61,4 +63,13 @@ public void checkoutToDate_invalidDate_throwsEmptyCommitException() throws Commi
         Date untilDate = TestUtil.getDate(2015, Calendar.FEBRUARY, 6);
         GitChecker.checkoutToDate(config.getRepoRoot(), config.getBranch(), untilDate);
     }
+
+    @Test
+    public void checkoutToDate_emptyBranch_success() throws CommitNotFoundException {
+        Path newFile = Paths.get(config.getRepoRoot(), "newFile.java");
+        Assert.assertTrue(Files.exists(newFile));
+        Date untilDate = TestUtil.getDate(2018, Calendar.FEBRUARY, 6);
+        GitChecker.checkoutToDate(config.getRepoRoot(), DEFAULT_BRANCH, untilDate);
+        Assert.assertFalse(Files.exists(newFile));
```

Comment:
This test doesn't seems to be any useful. 

If you're forcing the test to use the same code as its internal implementation (e.g. `DEFAULT_BRANCH`), you're doing double job and also not checking whether `DEFAULT_BRANCH` would be called and used in actual scenario.